next.config.js

```diff
@@ -1,5 +1,16 @@
 const { RetryChunkLoadPlugin } = require("webpack-retry-chunk-load-plugin");
 
+const self = "'self' https://*.replay.io wss://*.replay.io";
+const csp = `
+  frame-ancestors ${self};
+  default-src ${self};
+  style-src ${self} 'unsafe-inline';
+  script-src ${self} 'unsafe-eval' data: blob: https://*.stripe.com https://*.lr-ingest.io https://*.intercom.io https://*.intercomcdn.com https://*.launchdarkly.com;
+  connect-src ${self} https://webreplay.us.auth0.com https://*.launchdarkly.com/ https://*.stripe.com https://*.sentry.io https://*.intercom.io wss://*.intercom.io;
```

Comment:
Have we looked at the docs for these services, or are these based on observations of network traffic? For example https://www.intercom.com/help/en/articles/3894-using-intercom-with-content-security-policy include `https://uploads.intercomusercontent.com`, among others.

I'd guess Sentry and other services have official docs for what they expect.

Reply:
I'll check the docs. There were all based on observations.